diff --git a/postgres-appliance/Dockerfile b/postgres-appliance/Dockerfile
index 8710b44..399fd42 100644
--- a/postgres-appliance/Dockerfile
+++ b/postgres-appliance/Dockerfile
@@ -21,16 +21,6 @@ RUN export DEBIAN_FRONTEND=noninteractive \
     # Disable unwanted cron jobs
     && rm -fr /etc/cron.??* \
     && truncate --size 0 /etc/crontab \
-\
-    && if [ "$DEMO" != "true" ]; then \
-        # Required for wal-e
-        apt-get install -y pv lzop \
-        # install etcdctl
-        && ETCDVERSION=2.3.8 \
-        && curl -L https://github.com/coreos/etcd/releases/download/v${ETCDVERSION}/etcd-v${ETCDVERSION}-linux-amd64.tar.gz \
-                | tar xz -C /bin --strip=1 --wildcards --no-anchored etcdctl etcd; \
-    fi \
-\
     # Cleanup all locales but en_US.UTF-8 and optionally specified in ADDITIONAL_LOCALES arg
     && find /usr/share/i18n/charmaps/ -type f ! -name UTF-8.gz -delete \
     # Prepare find expression for locales
@@ -362,10 +352,8 @@ RUN export DEBIAN_FRONTEND=noninteractive \
             /usr/lib/postgresql/*/bin/pltcl_* \
     && find /var/log -type f -exec truncate --size 0 {} \;

-# Install patroni, wal-e and wal-g
+# Install patroni
 ENV PATRONIVERSION=2.0.2
-ENV WALE_VERSION=1.1.1
-ENV WALG_VERSION=v0.2.19
 RUN export DEBIAN_FRONTEND=noninteractive \
     && set -ex \
     && BUILD_PACKAGES="python3-pip python3-wheel python3-dev git patchutils binutils" \
@@ -382,24 +370,14 @@ RUN export DEBIAN_FRONTEND=noninteractive \
 \
     && if [ "$DEMO" != "true" ]; then \
         EXTRAS=",etcd,consul,zookeeper,aws" \
-        && curl -sL https://github.com/wal-g/wal-g/releases/download/$WALG_VERSION/wal-g.linux-amd64.tar.gz \
-                | tar -C /usr/local/bin -xz \
-        && strip /usr/local/bin/wal-g \
         && apt-get install -y python3-etcd python3-consul python3-kazoo python3-meld3 \
                         python3-boto python3-gevent python3-greenlet python3-cachetools \
                         python3-rsa python3-pyasn1-modules python3-swiftclient \
 \
         && find /usr/share/python-babel-localedata/locale-data -type f ! -name 'en_US*.dat' -delete \
 \
-        && pip3 install filechunkio wal-e[aws,google,swift]==$WALE_VERSION \
-                'git+https://github.com/zalando/pg_view.git@master#egg=pg-view' \
-\
-        # Revert https://github.com/wal-e/wal-e/commit/485d834a18c9b0d97115d95f89e16bdc564e9a18, it affects S3 performance
-        && curl -sL https://github.com/wal-e/wal-e/commit/485d834a18c9b0d97115d95f89e16bdc564e9a18.diff \
-                | patch -d /usr/local/lib/python3.6/dist-packages/wal_e -Rp2 \
-        # https://github.com/wal-e/wal-e/issues/318
-        && sed -i 's/^\(    for i in range(0,\) num_retries):.*/\1 100):/g' \
-                    /usr/lib/python3/dist-packages/boto/utils.py; \
+        && pip3 install filechunkio \
+                'git+https://github.com/zalando/pg_view.git@master#egg=pg-view'; \
     fi \
     && pip3 install "git+https://github.com/zalando/patroni.git@2.0/bugfixes#egg=patroni[kubernetes$EXTRAS]" \
 \
